docs/workflow_solutions/getting_containers.md

```diff
@@ -287,6 +287,66 @@ python python_test.py
+
+1. Specific licensing restrictions to ensure compliance with licensing terms.
+1. Compatibility/dependency issues: For example, when software require package or kernel versions that are incompatible with Cheaha, or have other system-specific dependencies.
+1. Software not available on Docker Hub: When you need to containerize software and its dependencies for sharing within your group or organization, and you are unable to get one anywhere open-source containers can be found (DockerHub, etc).
+
+### Where Can I Find UAB Research Computing Community Containers
+
+Our community containers are housed in the university’s GitLab repository. You can access them at <https://gitlab.rc.uab.edu/rc-data-science/community-containers>
+
+Each container is organized into a folder within the registry repository, making it easy to locate specific software or environments. Every container also includes a README file that provides documentation on its purpose, which software is included, and instructions for setting it up. If you have a container you would like to add to the container registry, please contact us at <support@listserv.uab.edu>, and we will gladly facilitate its inclusion.
+
+### How to Run UAB RC Community Containers
+
+To use a container from the registry, navigate to the preferred container. On the left navigation pane, you will see an option "Deploy", select this and a list of options will show up.
+
+![Accessing a container in a GitLab container registry](./images/gitlab-deploy-container.png)
+
+Click on "Container Registry" and then click the container, usually in the format "`containername/containername`", this would open a new page displaying the tag(s) (usually the `Commit SHA`). Use the copy icon next to the tag to copy the container's image path.
+
+![Steps for copying an image path in the GitLab Container Registry](./images/gitlab-container-tag.png)
+
+The next set of instructions will guide you through how to complete the install process for the container on Cheaha or on other platforms.
+
+#### Run UAB RC Community Containers on Cheaha using Singularity
+
+On Cheaha, you can pull a container using Singularity by running the `singularity pull` command, an example is shown below. Ensure you are in the preferred location you would like to use the container from.
+
+``` bash
+singularity pull <preferredName.sif> docker://gitlab.rc.uab.edu:<0000>/rc-data-science/community-containers/<name of container/name of container:tag>
+```
+
+In the above command, `<preferredName.sif>` is a file name placeholder you should replace with your preferred container image filename. For example you can rename the file as "`alphafold3.sif`". The `<0000>` is a placeholder for the specific port for our GitLab container registry (at this time it is 4567). For the `<name of container/name of container:tag>` placeholder, you will need to replace this placeholder with the specific path with the image name and tag of the container you want to pull.
```

Comment:
Let's make our port number more prominent, rather than as an aside in a parenthetical.